1.14.5/bullseye/Dockerfile

```diff
@@ -47,13 +49,15 @@ RUN set -ex && ARCHITECTURE=$(dpkg --print-architecture) \
     && if [ "${ARCHITECTURE}" = "armhf" ]; then RLS_ARCH=arm && RLS_LIB=gnueabihf; fi \
     && if [ "${ARCHITECTURE}" = "i386" ]; then RLS_ARCH=i686-pc; fi \
     && if [ "${RLS_ARCH}" = "" ]; then echo "Could not determine architecture" >&2; exit 1; fi \
-    && RLS_FILE_NAME=dogecoin-${RLS_VERSION}-${RLS_ARCH}-${RLS_OS}-${RLS_LIB}.tar.gz \
-    && wget ${RLS_LOCATION}/${RLS_FILE_NAME} \
-    && gitian/bin/gverify --no-markup -d sigs -r ${SIG_PATH} ${DESCRIPTOR_PATH} \
-       | grep OK | shuf -n 1 | sed s/:.*// > random_signer.txt \
-    && grep ${RLS_FILE_NAME} sigs/${SIG_PATH}/$(cat random_signer.txt)/*assert | sha256sum -c \
-    && grep ${RLS_FILE_NAME} SHASUMS | sha256sum -c \
-    && mv ${RLS_FILE_NAME} dogecoin.tar.gz
+    && RLS_FILE_NAME="dogecoin-${RLS_VERSION}-${RLS_ARCH}-${RLS_OS}-${RLS_LIB}.tar.gz" \
+    && wget -q --show-progress --progress=bar:force:noscroll "${RLS_LOCATION}/${RLS_FILE_NAME}" \
+    && GITIAN_OUTPUT=$(gitian/bin/gverify --no-markup -d sigs -r "${SIG_PATH}" "${DESCRIPTOR_PATH}") || true \
```

Comment:
I see why you used `|| true`, gverify exit with 1 whenever a signature in the list is invalid, so probably always.

The script must have a `RANDOM_SIGNER`, it will stop without it. Related to my previous comment, we could use `set -u` and remove variable check/error message.
A bit annoying this gverify behavior.

Reply:
It's just that sigs got merged without checking, so there is a set of signatures that causes mismatch. This can happen also when there is contention, so it's good that it exits with an error. Just not entirely cool for our use case, but I don't think it should change for generic gverify. 

As an alternative, we can script a gitian verification process that suits our needs better, but with it nearing end-of-life, we may as well settle on negating the error exit for now.

Reply:
> It's just that sigs got merged without checking, so there is a set of signatures that causes mismatch. This can happen also when there is contention, so it's good that it exits with an error. Just not entirely cool for our use case, but I don't think it should change for generic gverify.
> 
> As an alternative, we can script a gitian verification process that suits our needs better, but with it nearing end-of-life, we may as well settle on negating the error exit for now.

i think this is fine but a note that fdov, @AbcSxyZ and my keys got merged after v1.14.5 tag so it's only pulling langerhans, michi, rnicoll, stapler117, whit and your keys:

that's why fdov, @AbcSxyZ and mine get:
```
gpg: Can'\''t check signature: No public key
xanimo: BAD SIGNATURE
```
because our key's aren't imported in `find dogecoin/contrib/gitian-keys -name '*.pgp' -exec gpg --import '{}' +` line. if we pulled from `-b 1.14.6-dev` we're included/can verify. regardless since nearing eol and having 4/6 gitian signers for v1.14.5 and only needing 1 to gverify against is fine and works to verify authenticity.